google/ggrc-core
# GGRC-4499 Fix automatic migrations

# Issue description

Deploying a 1.4 release, and running the migrations from the maintenance window turns of maintenance mode and never shuts it off.

The logs show that the migrations finish, but migration status on the maintenance window always shows "in progress"

# Steps to test the changes

Deploy the changes to any appspot instance and check the logs.

Or run migrations locally in the maintenance window using launch_gae_ggrc

Also verify that all other ways of running the migrations still work.

# Solution description

Wrap the migration into  an app session since the last commit complained about missing it. I am not fully sure why this started though, just that it does seem to fix it.

If anyone can find the actual cause of why this bug just started appearing and suggest a better solution I would be quite happy.


# Sanity checklist




# PR Review checklist
